nemo/collections/tts/modules/fastpitch.py

```diff
@@ -188,8 +194,8 @@ def __init__(
         else:
             self.speaker_emb = None
 
+        self.min_token_duration = 1
```

Comment:
In theory it shouldn't since the aligner produces duration prediction targets with a minimum of 1. Otherwise, I am not sure how I would test that it impacts nothing.

As mentioned in the PR description, we could also make this a config (defaulting to 0) if we want to be very safe.